test/parallel/test-icu-data-dir.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
-require('../common');
+const common = require('../common');
+if (!(common.hasIntl && common.hasSmallICU)) {
```

Comment:
Is `common.hasSmallICU` a thing? I'm only finding `process.binding('config').hasSmallICU` with a quick search.